doc/source/user_guide/timeseries.rst

```diff
@@ -1271,6 +1271,21 @@ frequencies. We will refer to these aliases as *offset aliases*.
     "U, us", "microseconds"
     "N", "nanoseconds"
 
+.. note::
+
```

Comment:
umm w/o explicit examples I am not sure how useful this note is, so pls add these